### PR TITLE
Add search by date parameter to Encounter

### PR DESCRIPTION
**Pull request by viktorhegedusepam**
Description
----

Screenshot 2021-06-09 at 9 29 46

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screenshot 2021-06-16 at 8 35 57

### CODE REVIEW

File: content/millennium/r4/encounters/encounter.md [hunk 1]
```diff
@@ -87,12 +87,29 @@ _Implementation Notes_
...
+------------|---------------------------------------------|---------------|-------------------------------------------------------------------------------------------------------------------
  `_id`      | This or `patient` or `subject` or `account` | [`token`]     | The logical resource id associated with the Encounter. Example: `7891`
  `patient`  | This or `subject` or `account` or `_id`     | [`reference`] | The patient present at the encounter. Example: `12345`
  `subject`  | This or `patient` or `account` or `_id`     | [`reference`] | The patient present at the encounter. Example: `subject=Patient/12345` or `subject:Patient=12345`
  `account`  | This or `patient` or `subject` or `_id`     | [`reference`] | The account associated with the encounters. Example: `F703726`
- [`_count`] | No                                          | [`number`]    | The maximum number of results to return.
+ `date`     | N                                           | [`dateTime`]  | Datetime range into which the encounter's period datetime falls. Must be prefixed by 'ge', 'gt', 'le' or 'lt'.
```
Comment (Sanvenky) [inline content/millennium/r4/encounters/encounter.md:95]: make it No to be consistant

**Review by Sanvenky — CHANGES_REQUESTED** [review]

File: content/millennium/r4/encounters/encounter.md (hunk 1 above)
Comment (viktorhegedusepam) [reply content/millennium/r4/encounters/encounter.md]: Fixed.

**Review by Sanvenky — APPROVED** [review]
+1 after conflict resolved.

**Review by aki-mathur — APPROVED** [review]

File: content/millennium/r4/encounters/encounter.md [hunk 2]
```diff
@@ -95,9 +95,10 @@ _Implementation Notes_
...
  `account`    | This or `patient` or `subject` or `_id`     | [`reference`] | The account associated with the encounters. Example: `F703726`
- [`_count`]   | No                                          | [`number`]    | The maximum number of results to return.
+ `date`       | No                                          | [`dateTime`]  | Datetime range into which the encounter's period datetime falls. Must be prefixed by 'ge', 'gt', 'le' or 'lt'.
+[`_count`]    | No                                          | [`number`]    | The maximum number of results to return.
  `status`     | No                                          | [`token`]     | The status of the encounter. Example: `planned`
-`_revinclude` | No                                          | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
+ `_revinclude`| No                                          | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
```
Comment (Sanvenky) [inline content/millennium/r4/encounters/encounter.md:101]: can you correct this space as before for  `_revinclude`?

File: content/millennium/r4/encounters/encounter.md [hunk 3]
```diff
@@ -95,9 +95,10 @@ _Implementation Notes_
  `patient`    | This or `subject` or `account` or `_id`     | [`reference`] | The patient present at the encounter. Example: `12345`
  `subject`    | This or `patient` or `account` or `_id`     | [`reference`] | The patient present at the encounter. Example: `subject=Patient/12345` or `subject:Patient=12345`
  `account`    | This or `patient` or `subject` or `_id`     | [`reference`] | The account associated with the encounters. Example: `F703726`
- [`_count`]   | No                                          | [`number`]    | The maximum number of results to return.
+ `date`       | No                                          | [`dateTime`]  | Datetime range into which the encounter's period datetime falls. Must be prefixed by 'ge', 'gt', 'le' or 'lt'.
+[`_count`]    | No                                          | [`number`]    | The maximum number of results to return.
```
Comment (Sanvenky) [inline content/millennium/r4/encounters/encounter.md:99]: i think there was a spce before [`_count`]

File: content/millennium/r4/encounters/encounter.md (hunk 2 above)
Comment (viktorhegedusepam) [reply content/millennium/r4/encounters/encounter.md]: Fixed

File: content/millennium/r4/encounters/encounter.md (hunk 3 above)
Comment (Sanvenky) [reply content/millennium/r4/encounters/encounter.md:99]: align this with date

Comment (Sanvenky) [reply content/millennium/r4/encounters/encounter.md:99]: may need to add 2 spaces

Comment (viktorhegedusepam) [reply content/millennium/r4/encounters/encounter.md]: Fixed.

**Comment by Zonglin123** [comment]
Screen Shot 2021-06-18 at 12 46 09 PM

**Pull request merged**